languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/en-GB/grammar.xml

```diff
@@ -534,6 +534,55 @@ USA
             <example type="triggers_error">Explore releases from Ait! at Discogs.</example>
         </rule>
 
-    </category>
+        </category>
+
+        <rule id="YOUR_YOURE" name="Your/You're Usage">
+            <pattern>
+                <token>your</token>
+            </pattern>
+            <message>Did you mean <suggestion>you're</suggestion>?</message>
+            <example correction="you're">I think <marker>your</marker> going to love it.</example>
+        </rule>
+
+        <rule id="WORD_COLLOCATION_1" name="Inappropriate Word Collocation">
+            <pattern>
+                <token>strong</token>
+                <token>believe</token>
+            </pattern>
+            <message>Consider using <suggestion>strongly believe</suggestion> instead of 'strong believe'. "Strongly" is the adverb form of "strong" and is used to modify verbs like "believe."</message>
+            <example correction="strongly believe">I strong believe in your abilities.</example>
+        </rule>
+
+        <rule id="WORD_COLLOCATION_3" name="Inappropriate Word Collocation">
+            <pattern>
+                <token>fast</token>
+                <token>drive</token>
+            </pattern>
+            <message>Consider using <suggestion>fast driving</suggestion> instead of 'fast drive'. "Driving" is the gerund form of the verb "drive" and is used after an adjective like "fast" to describe the act of driving.</message>
+            <example correction="fast driving">He was caught for fast drive.</example>
+        </rule>
+
+        <rule id="WORD_COLLOCATION_2" name="Inappropriate Word Collocation">
+            <pattern>
+                <token>fast</token>
+                <token>meal</token>
+            </pattern>
+            <message>Consider using <suggesiton>fast food</suggestion> instead of 'fast meal'. "Fast food" is a specific type of food that is prepared and served quickly, whereas "fast meal" is not commonly used.</message>
+            <example correction="fast food">Let's grab some fast meal.</example>
+        </rule>
+
+        <rule id="Singular_Subject_Verb_Agreement" name="Singular Subject-Verb Agreement">
+            <pattern>
+                <token postag="DT" min="0" max="1"/>
+                <token postag="NN">dog</token>
```

Comment:
This rule will apply only when the subject is `dog` and the verb is `run`. Could you think of ways to make it more robust? (Just flagging this for you. As noted above, LT already has rules for subject-verb agreement issues, so it would be preferable to refine those rules than create new ones from scratch.)